tests/operators/test_email_operator.py

```diff
@@ -57,6 +57,6 @@ def _run_as_operator(self, **kwargs):
         task.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE)
 
     def test_execute(self):
-        with conf_vars({('email', 'EMAIL_BACKEND'): 'tests.operators.test_email_operator.send_email_test'}):
+        with conf_vars({('email', 'email_backend'): 'tests.operators.test_email_operator.send_email_test'}):
```

Comment:
Just to confirm: we are setting these as lowercase because that is how it is defined in the config, but reading this as `conf.get('email', 'EMAIL_BACKEND')` still works, right?

Reply:
That's correct, we have explicit `lower` method call on both key and section in the `get` method. Only `as_dict` returns case sensitive keys and sections. I added an extra test case as proof.

It is not consistent right now, do we want to make `get` case sensitive as well?

Reply:
Probably not, no as that would break a _lot_ of code, and reading is almost universally safe to be case-insensitive.